lib/improver/units.py

```diff
@@ -62,10 +62,55 @@
     "projection_y_coordinate": {"unit": "m"},
     "percentile": {"unit": "%"},
     "probability": {"unit": "1"},
+    "realization": {"unit": "1", "dtype": np.int32},
+    "height": {"unit": "m"},
+    "velocity": {"unit": "m s-1"},
     # standard diagnostics and suitable substrings (alphabetised for clarity)
+    "air_temperature_lapse_rate": {"unit": "K m-1"},
+    "cloud": {"unit": "1"},
     "fall_rate": {"unit": "m s-1"},
+    "falling_snow_level": {"unit": "m"},
+    "humidity": {"unit": "1"},
     "lapse_rate": {"unit": "K m-1"},
+    "orographic_enhancement": {"unit": "m s-1"},
     "precipitation_rate": {"unit": "m s-1"},
+    "pressure": {"unit": "Pa"},
     "temperature": {"unit": "K"},
-    "thickness": {"unit": "m"}
+    "thickness": {"unit": "m"},
+    "ultraviolet_flux": {"unit": "W m-2"},
+    "ultraviolet_index": {"unit": "1"},
+    "visibility_in_air": {"unit": "m"},
+    "weather_code": {"unit": "1", "dtype": np.int32},
+    "wind_from_direction": {"unit": "degrees"},
+    "wind_speed": {"unit": "m s-1"},
+    "wind_to_direction": {"unit": "degrees"},
+    # ancillary diagnostics
+    "alphas": {"unit": "1"},
+    "grid_with_halo": {"unit": "no_unit"},
+    "emos_coefficients": {"unit": "1"},
+    "surface_altitude": {"unit": "m"},
+    "land_binary_mask": {"unit": "1", "dtype": np.int32},
+    "radar_coverage_mask": {"unit": "1"},
```

Comment:
This is an integer value.  Doesn't hurt to keep it `int16` if numpy supports that; otherwise `np.int32` should be the standard I think.

Reply:
Removed from dictionary and set input radar coverage mask to int16 type; it is float32 type in our current acceptance test data.